src/js/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -34,18 +34,17 @@ const Element = Scroll.Element;
 class DocumentRequestPage extends React.Component {
   componentDidMount() {
     this.props.resetUploads();
-    if (this.props.trackedItem) {
-      document.title = `Request for ${this.props.trackedItem.displayName}`;
-    } else {
-      document.title = 'Document Request';
-    }
+    document.title = `Request for ${this.props.trackedItem.displayName}`;
     if (!this.props.loading) {
       setUpPage();
     } else {
       scrollToTop();
     }
   }
   componentWillReceiveProps(props) {
+    if (props.loading && !!props.trackedItem) {
+      this.props.router.push(`/your-claims/${this.props.params.id}/status`);
```

Comment:
I think we want to use `replace` instead of `push`. Otherwise, using the back button will get someone stuck.